presto-main/src/main/java/com/facebook/presto/type/TimeOperators.java

```diff
@@ -196,4 +186,14 @@ public static boolean isDistinctFrom(
         }
         return notEqual(left, right);
     }
+
+    private static long getTimeInstantInCurrentDay(long value)
```

Comment:
Using SqlType annotations in functions that are not meant to be parsed by annotation parser seems like breaking code conventions. It doesn't meter anyway - this function does not exists any longer.